opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/TracingHandlerInterceptor.java

```diff
@@ -53,12 +53,16 @@ public TracingHandlerInterceptor(Tracer tracer, List<HandlerInterceptorSpanDecor
         this.decorators = new ArrayList<>(decorators);
     }
 
+    static boolean hasSpanStarted(HttpServletRequest httpServletRequest) {
```

Comment:
Can we change the name of this method to something like `notTraced` with a comment what is happening?